pkgs/by-name/ff/ffts/package.nix

```
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  git,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ffts";
+  version = "unstable-2019-03-19";
+
+  src = fetchFromGitHub {
+    owner = "linkotec";
+    repo = "ffts";
+    rev = "2c8da4877588e288ff4cd550f14bec2dc7bf668c";
+    hash = "sha256-Cj0n7fwFAu6+3ojgczL0Unobdx/XzGNFvNVMXdyHXE4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+  ];
+  buildInputs = [ ];
+
+  cmakeFlags = [
+    "-DCURRENT_GIT_VERSION=${lib.substring 0 7 src.rev}"
+    "-DENABLE_SHARED=ON"
+    "-Wno-deprecated"
```

```suggestion
    "-Wno-deprecated"
```
IMHO, we shouldn't remove build warnings.